src/os_posix/os_alloc.c

```diff
@@ -9,6 +9,27 @@
 #include "wt_internal.h"
 
 /*
+ * On systems with poor default allocators for allocations greater than 16 KB,
+ * we provide an option to use TCMalloc explicitly.
+ * This is important on Windows which does not have a builtin mechanism
+ * to replace C run-time memory management functions with alternatives.
+ */
+#ifdef HAVE_LIBTCMALLOC
+#include <gperftools/tcmalloc.h>
```

Comment:
@keithbostic Thanks. Your change is correct.  The `google` header is deprecated, the `gperftools` header is the right one.